src/OpenIddict.Security/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
```

Comment:
System. Reflection? I thought you added it. I rebased on your changes and had a conflict. Np.